Examples/Tests/field_probe/analysis_field_probe.py

```diff
@@ -24,11 +24,12 @@
 
 # Open data file
 df = pd.read_csv(filename, sep=' ')
-df = df.sort_values(by=['[2]part_x_lev0-(m)'])
+df.columns = df.columns.str.replace('#','')
```

Comment:
To make this efficient, maybe stop after first match? https://pandas.pydata.org/docs/reference/api/pandas.Series.str.replace.html
```suggestion
df.columns = df.columns.str.replace('#', '', n=1)
```

Reply:
You can also use https://pandas.pydata.org/docs/reference/api/pandas.Series.str.removeprefix.html
```suggestion
df.columns = df.columns.str.removeprefix('#')
```